src/officehours_ui/templates/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-Agent: *
+Disallow: /
```

Comment:
@ssciolla Do you think we should have a Disallow here? I guess this site really has nothing to index anyway and it's not something for the public, so seems like it would be fine, but this is a new file.